inf-clojure.el

```diff
@@ -642,10 +648,11 @@ it (default is value of `inf-clojure-*-cmd').  Runs the hooks
 from `inf-clojure-mode-hook' (after the `comint-mode-hook' is
 run).
 \(Type \\[describe-mode] in the process buffer for a list of commands.)"
-  (interactive (list (completing-read "Clojure startup command: "
-                                      (mapcar #'cdr inf-clojure-startup-forms)
-                                      nil
-                                      'confirm-after-completion)))
+  (interactive (list (or inf-clojure-custom-startup
+                         (completing-read "Clojure startup command: "
```

Comment:
I guess this is more of a "REPL startup command", although in the case of connecting it's different, so I feel the message is a bit confusing.

Reply:
Maybe we can drop "Clojure" here?